src/Websites/Sales/Directory.Build.props

```diff
@@ -3,7 +3,7 @@
     <PropertyGroup>
         <BlazorMode>BlazorServer</BlazorMode>
         <!-- You can use either BlazorServer or BlazorWebAssembly or BlazorHybrid -->
-        <WebAppDeploymentType>DefaultDeploymentType</WebAppDeploymentType>
+        <WebAppDeploymentType>SSR</WebAppDeploymentType>
```

Comment:
Revert this

Reply:
we talked about it and decided to go with SSR from now on.